website/frontend/src/pages/OurProducts/CalibrationPage.js

```diff
@@ -0,0 +1,98 @@
+              Calibration enhances the quality and reliability of air quality data from low-cost sensors
+              </p>
+            </div>
+            <div className="overlap-section">
+              <div className="lapping-left card" /*card-sm larger-top*/  style={{ backgroundColor: '#FFFCE1' }}>
+                <h5>Cost effective and accessible</h5>
+                <p>
+                AirQalibrate is a Machine Learning based calibration tool that eliminates the need for reference grade monitor 
+                or on-site monitor calibration.<br />
+                <br />
+                It enables users without access to reference grade monitors or technical expertise 
+                to calibrate data from their low-cost monitors and improve performance and build trust in the air quality network.<br />
+                <br />
+                This reduces operational costs involved in monitoring thereby enabling the expansion of monitoring systems 
+                to create a dense air quality monitoring and management network.
+                </p>
+              </div>
+              <div className="lapping-left image" id="section-1">
+                <img src={Section1Image} alt="" />
+              </div>
+            </div>
+          </div>
+
+          <div className="grid-wrapper section">
+            <div className="overlap-section">
+              <div className="lapping-right card" style={{ backgroundColor: '#E6E6FA', marginTop: '30px' }}>
+                <h5>Calibrate your data</h5>
+                <p>
+                Upload a CSV file containing your low-cost sensor  PM<sub>2.5</sub> and PM<sub>10</sub>  data, follow a few simple steps and calibrate your data. 
+                The results are automatically downloaded when the calibration process is completed.
+                 <br/>
+                <a href='https://airqalibrate.airqo.net/' target='_blank' className='download-button'> Calibrate your data <OpenInNew /></a>
+                </p>
+
+              </div>
+              <div className="lapping-right image" id="section-2">
+                <img src={Section2Image} alt="" />
+              </div>
+            </div>
+          </div>
+
+          <div className="grid-full column section landscape-section">
+            <div className="column smaller-width mx-wdth-800px">
+              <h2 className="left title">
+                <span className="blue">Simple </span> user interface
+              </h2>
+              <p className="right ">
+              Our calibration tool features a user-friendly interface that simplifies the calibration process. 
+              Even without technical expertise, you can easily navigate the tool and calibrate the data from air quality monitors. 
+              </p>
+              <NavTab text="Calibration guide" path="https://wiki.airqo.net/#/calibration/ml_based_approach" hideArrow filled />
```

Comment:
@geneviva Thanks for the changes so far.
This link here should open in a new tab (view implementation in API Page file).